components/ui/Tag.jsx

```diff
@@ -0,0 +1,11 @@
+const Tag = ({ content }) => {
+  return (
+    <div>
+      <p className="font-neuf text-lg text-primaryGray-400 font-normal -tracking-wide bg-primaryGray-200 rounded-base px-2.5 py-sm  w-fit">
```

Comment:
`font-normal` className is not actually needed since it doesn't change anything on the text, it's good to remove it.